dynamiqs/utils/vectorization.py

```diff
@@ -214,4 +214,4 @@ def slindbladian(H: Tensor, L: Tensor) -> Tensor:
     Returns:
         _(..., n^2, n^2)_ Lindbladian superoperator.
     """
-    return -1j * (spre(H) - spost(H)) + sdissipator(L).sum(-3)
+    return -1j * (spre(H) - spost(H)) + sdissipator(L).sum(0)
```

Comment:
I feel it's more natural to have the multiple jump ops dimension on the left because one will define all jump ops and then put them in a list. People will do so for `mesolve` for instance so even in term of API consistency it makes more sense to use `L _(N, ..., n, n)`.
I don't want to  argue on this so you choose.

Reply:
In the meantime I made a private function for this

Reply:
For consistency with `mesolve`, we should modify the API of `slindbladian` to accept a list of jump operators, so of format `list[Tensor]`, where each Tensor can be batched. And in this case, I agree with @abocquet.